### PR TITLE
Implement client for feature service

### PR DESCRIPTION
**Pull request by c-w**
This will be required for a few of the GraphQL resolvers, e.g. `MessageSchema.byBbox`, `TilesSchema.fetchPlacesByBbox` and `EdgesSchema.locations`.

~~Should be merged after #21.~~

### CODE REVIEW

**Review by Smarker — APPROVED** [review]
LGTM but maybe you'd want to put modules.export at the top so that we can easily see all the functions we have available to us?

File: src/clients/locations/FeatureServiceClient.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+function formatBboxUri(north, west, south, east) {
+  return `http://${apiHost}/features/bbox/${north}/${west}/${south}/${east}`;
+}
+
+function formatPointUri(latitude, longitude) {
+  return `http://${apiHost}/features/point/${latitude}/${longitude}`;
+}
+
+function formatNameUri(names) {
+  return `http://${apiHost}/features/name/${names.join(',')}`;
```
Comment (jcjimenez) [inline src/clients/locations/FeatureServiceClient.js]: Do we need to URL escape all these values?

**Review by jcjimenez — APPROVED** [review]
LGTM with question.

**Comment by c-w** [comment]
@Smarker Always declaring the exports at the bottom of the file also makes it easy to find them and has the benefit that it always works, even for exports that are declared as variables instead of functions ([read more](https://stackoverflow.com/q/23791749/3817588)).

File: src/clients/locations/FeatureServiceClient.js (hunk 1 above)
Comment (c-w) [reply src/clients/locations/FeatureServiceClient.js]: Done in c381784.

**Comment by Smarker** [comment]
Alright, good point. I agree.

**Pull request merged**